app/views/users/show.html.erb

```diff
@@ -10,9 +10,10 @@
 <div class="user_show">
 	<% @user.questions.each do |question| %>
 	<ul>
-	  <li><%= question.title %></li>
+	  <li><%= link_to "#{question.title}", question_path(question.id) %></li>
```

Comment:
just FYI instead of question_path(question.id) it can be just be:

<%= link_to "#{question.title}", question %>

nice shortcut rails let you do